### PR TITLE
version conflicts

### PR DESCRIPTION
**Pull request by web-dave**
App-shell seems to generate a totally version conflict. Can't solve it. And angular material needs animation module

### CODE REVIEW

**Comment by web-dave** [comment]
I think appshell and material are in beta very much.

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # compiled output
 /dist
 /tmp
+/build
```
Comment (webmaxru) [inline .gitignore]: I commit `build` folder by intention - this `pwa-workshop` is for "generic" JavaScript workshops (unlike `pwa-workshop-angular`), without need to have `angular-cli` installed.

File: package.json [hunk 2]
```diff
@@ -11,17 +11,18 @@
     "e2e": "ng e2e",
     "build-prod": "ng build --prod",
     "generate-sw": "sw-precache --config=./sw-precache-config.js --verbose",
-    "build-full": "npm run build-prod && npm run generate-sw"
+    "build-full": "npm run build-prod && npm run generate-sw",
+    "rest": "cd ../pwa-workshop-api && npm start"
```
Comment (webmaxru) [inline package.json]: I'd prefer to not to bind these two repos so tight

**Review by webmaxru — CHANGES_REQUESTED** [review]
Thanks for contributing! Please have a look at my comments.

File: .gitignore (hunk 1 above)
Comment (web-dave) [reply .gitignore]: That makes totally sense. I'll fix it today.

File: package.json (hunk 2 above)
Comment (web-dave) [reply package.json]: Ok. I love this way but I'm fine with it. I'll fix it Today

**Comment by web-dave** [comment]
I did the changes as requested.

**Comment by web-dave** [comment]
I'm very excited about the workshop I'll do on Friday.

**Pull request merged**